packages/lesswrong/components/moderationTemplates/ModerationTemplatesPage.tsx

```diff
@@ -4,24 +4,39 @@ import { ModerationTemplates } from "../../lib/collections/moderationTemplates";
 import { userCanDo } from "../../lib/vulcan-users";
 import { useCurrentUser } from "../common/withUser";
 import {useMulti} from "../../lib/crud/withMulti";
+import { ALLOWABLE_COLLECTIONS } from '../../lib/collections/moderationTemplates/schema';
+import classNames from 'classnames';
 
 const styles = (theme: ThemeType): JssStyles => ({
   form: {
     border: theme.palette.border.commentBorder,
     padding: 12,
     paddingLeft: 16,
     background: theme.palette.panelBackground.default,
+  },
+  filter: {
+    ...theme.typography.body2,
+    padding: 8,
+    border: theme.palette.border.commentBorder,
+    borderRadius: 2,
+    cursor: "pointer",
+    marginRight: 8,
+    background: theme.palette.panelBackground
+  },
+  filterSelected: {
+    backgroundColor: theme.palette.grey[200]
   }
 });
 
 //a page for creating and viewing moderation templates
 export const ModerationTemplatesPage = ({classes}: {
   classes: ClassesType,
 }) => {
-  const { WrappedSmartForm, SingleColumnSection, SectionTitle, ModerationTemplateItem, BasicFormStyles, Loading } = Components
+  const { WrappedSmartForm, SingleColumnSection, SectionTitle, ModerationTemplateItem, BasicFormStyles, Loading, Row, ToCColumn, TableOfContents } = Components
   
   const currentUser = useCurrentUser();
   const [showDeleted, setShowDeleted] = useState<boolean>(false);
+  const [filter, setFilter] = useState<string|null>(null);
```

Comment:
I think this can be more narrowly typed as `TemplateType | null`, rather than `string | null`